src/css/storyTelling.css

```diff
@@ -107,28 +107,8 @@ fieldset ul {
     border-color: #CCC;
 }
 
-.sjrk-st-button-text-block-container {
-    grid-column: 1;
-}
```

Comment:
The removal of these CSS classes is on purpose? Does it have to do with SJRK-190, or just unrelated clean-up?

Reply:
This was unrelated cleanup, yes. In hindsight, I should have made it under a NOJIRA commit at the very least.

Reply:
In fact, more accurately, it's cleanup after fixing a styling bug related to the button positioning that only manifested itself on smaller screens.